### PR TITLE
added conda package for scrapy-contrib-bigexporters

### PR DESCRIPTION
**Pull request by jornfranke**
Checklist
- [ X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ X] Source is from official source
- [ X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ X] If static libraries are linked in, the license of the static library is packaged.
- [X ] Build number is 0
- [ X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
Could you please fix the linters errors?
If you want to, you can try to use [Grayskull](https://github.com/conda-incubator/grayskull) to generate this recipe for you

It will give you something like
```yaml
{% set name = "scrapy-contrib-bigexporters" %}
{% set version = "0.3.0" %}


package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scrapy-contrib-bigexporters-{{ version }}.tar.gz
  sha256: 9438c5f6d76bb8d68d38f9472dddab7f4da21abce6da1d320aabdca840046288

build:
  number: 0
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv

requirements:
  host:
    - pip
    - python >=3.6
  run:
    - python >=3.6
    - scrapy >=2.4.0

test:
  imports:
    - tests
    - zuinnote
  commands:
    - pip check
  requires:
    - pip

about:
  home: https://github.com/zuinnote/scrapy-contrib-bigexporters
  summary: Scrapy exporter for Big Data formats
  license_file: PLEASE_ADD_LICENSE_FILE

extra:
  recipe-maintainers:
    - ADD_YOUR_GITHUB_ID_HERE

```

File: recipes/scrapy-contrib-bigexporters/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  sha256: 9438c5f6d76bb8d68d38f9472dddab7f4da21abce6da1d320aabdca840046288
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
```
Comment (marcelotrevisani) [inline recipes/scrapy-contrib-bigexporters/meta.yaml:24]: can you please remove all the comments?

**Comment by jornfranke** [comment]
sorry for my late reply and thank you got the chance to quickly check it.
I incorporated your feedback.

File: recipes/scrapy-contrib-bigexporters/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - scrapy >=2.4.0
+  run_constrained:
+    - fastparquet
```
Comment (dopplershift) [inline recipes/scrapy-contrib-bigexporters/meta.yaml:25]: What are you trying to accomplish with `run_constrained`? Without version numbers, it's not going to enforce anything. So either need to remove this section or add versions to do the constraining.

Comment (jornfranke) [reply recipes/scrapy-contrib-bigexporters/meta.yaml:25]: thank you. I tried to mark them as optional dependencies. I will add the version numbers

Comment (jornfranke) [reply recipes/scrapy-contrib-bigexporters/meta.yaml]: After reflection, I removed run_constrained

**Review by dopplershift — APPROVED** [review]

**Pull request merged**